### ISSUE TITLE
Laser turret is too underpowered, also crashes at upgrade 3

### ISSUE THREAD

**Issue by sloanonlinux** [reloZid/android-anuto#27]
It's fire rate is too slow, if I start off with a laser turret I will have monsters slipping by. It's second upgrade isn't very powerful either. The game locked up at upgrade 3.

This is a good game though, keep up the good work!

**Comment by mvdan** [comment]
I agree that it's a little underpowered. The only way laser towers are useful is if they are at maximum level and placed alongside a straight path. Even then they're not particularly powerful, either.

**Comment by bart9h** [comment]
Of course the lasers are supposed to be used on a straight path. They're not underpowered at all if used correctly. I always put three level-3 lasers on the first horizontal path, two on the left with first/locked, and one on the right with last/unlocked. The damage they do for one creep is low, but when the lane is full of creeps, it adds up nicely.

**Comment by jkufner** [comment]
Laser lvl3 works nicely around wave 25, but later is basically useless.

**Comment by gloryroad1** [comment]
The 2nd enhancement of the laser seems out of balance with the 2nd enhancements of the cannon and rocket, which though expensive in cost per damage, are hugely effective.  The 2nd laser enhancement zaps out in a straight line and seems to me to do little damage.  The 1st laser enhancement bends to hit multiple targets and seems much more effective for it's cost.  2nd enhancements of the cannon and rocket do dozens, if not hundreds, of megs of damage while the 2nd laser counts are still in the single megs.

**Comment by ranit8** [comment]
Suggestion: how about making Lasers strong against Blobs instead of Flyers, and the opposite change on Explosives? Lasers are useless in late-game because they are supposed to always fire in the same direction to maximize damage, and the Blobs, which are slow and stay longer on the same place, resist them.

The targets Lasers damage most are the Flyer (which cant be slowed) and the Healer (which requires many towers hitting a single target, and Lasers should be distributed along the straights).
